src/commands/web/spec/web.spec.ts

```diff
@@ -54,7 +72,34 @@ describe('sasjs web', () => {
 
       await expect(buildWebApp(new Command(`web -t viya`))).resolves.toEqual(0)
 
-      await expect(verifyFolder(webBuiltFiles)).resolves.toEqual(true)
+      await expect(verifyFolder(webBuiltFilesSASVIYA)).resolves.toEqual(true)
+    },
+    2 * 60 * 1000
+  )
+
+  it(
+    `should create web app with minimal template for target SAS9`,
+    async () => {
+      appName = `cli-test-web-minimal-${generateTimestamp()}`
+      await createTestMinimalApp(__dirname, appName)
```

Comment:
You shouldn't create new minimal app for each test suite

Reply:
Is this for performance?  Yeah I guess you could test both SAS 9 and Viya in the same test

Reply:
performing sas9 and sasviya builds on same app now